modules/optimization/.coverage

```diff
@@ -0,0 +1,11 @@
+# Coverage requirement configuration for moose, enforced in CIVET testing
```

Comment:
Remove file; add requirement to .coverage in moose root

Reply:
done.